backend/internal/handlers/webhooks/webhooks.go

```diff
@@ -87,6 +88,14 @@ func (s *WebHookService) PushEvent(c *fiber.Ctx) error {
 	return nil
 }
 
+func hasNamedPusher(pushEvent github.PushEvent) bool {
+	return pushEvent.Pusher != nil && pushEvent.Pusher.Name != nil
```

Comment:
using the name field can break things because the name field is optional on a github user. If you want their username, that is the field Login. It also doesn't make sense to require that the github user has a name set on github